### PR TITLE
feat: fetch data and provide to client with context

### LINKED ISSUE #11: fetch sample data in backend
Eventually we want to fetch the data in the backend by calling the Graasp API.

As a first step, we can retrieve the sample data in the backend so that it sends the complete markup ready to be displayed.

### PR DESCRIPTION
**Pull request by pyphilia**
close #11

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -3,50 +3,70 @@ import { StaticRouter } from 'react-router-dom';
...
+          ? `<script src="${assets.client.js}" defer></script>`
+          : `<script src="${assets.client.js}" defer crossorigin></script>`
+      }
+  </head>
+  <body>
+      <div id="root">${markup}</div>
+      <script>
+        // WARNING: See the following for security issues around embedding JSON in HTML:
+        // https://redux.js.org/recipes/server-rendering/#security-considerations
+        window.PRELOADED_STATE = ${JSON.stringify(data)?.replace(
```
Comment (juancarlosfarah) [inline src/server.js:49]: todo: ensure that this can only have data we allow

Comment (juancarlosfarah) [reply src/server.js:49]: You can leave this for a future PR.

File: src/server.js [hunk 2]
```diff
@@ -3,50 +3,70 @@ import { StaticRouter } from 'react-router-dom';
...
 </html>`,
-      );
-    }
-  });
+    );
+  }
+};
+
+const handleCollectionRender = (req, res) => {
+  const id = req.path.split('/')[2];
```
Comment (juancarlosfarah) [inline src/server.js:61]: Can we make this more robust?

**Review by juancarlosfarah — APPROVED** [review]
Looks good to me, but I have a question for the collection render method.

File: src/server.js (hunk 1 above)
Comment (pyphilia) [reply src/server.js:49]: address in issue #15

**Pull request merged**